### PR TITLE
Add capability to show multiple regression result sets 

### PR DESCRIPTION
**Pull request by rlamping**
**Issue: Add capability to show multiple regression result sets** 
Addition of code to display the coefficients (without intercept) of multiple regression result sets
Still to do: 
- documentation 
- adding examples

### CODE REVIEW

File: R/corrplot.R [hunk 1]
```diff
@@ -198,6 +198,10 @@
 #' @param win.asp Aspect ration for the whole plot. Value other than 1 is
 #'   currently compatible only with methods "circle" and "square".
 #'
+#' @param mreg Logical, whether matrix is a multiple regression representation
+#'  (TRUE) or something else (FALSE.) Default is FALSE as corrplot is initially
+#'  meant for linia
```
Comment (vsimko) [inline R/corrplot.R]: typo? "meant for linia" ...

File: R/corrplot.R [hunk 2]
```diff
@@ -297,6 +304,26 @@ corrplot <- function(corr,
...
+    #if (is.null(cl.lim)) {
+      # if the matrix is expected to be a multiple regression correlation matrix
+      # it MUST be within the interval [-1,1], although it can be overruled
+    #  cl.lim <- c(-1,1)
+    #}
+  }
+
+  # mreg code: catch corr for multiple regression
+  # cross out of ignored predictors
+  corrbefore <- corr
```
Comment (vsimko) [inline R/corrplot.R]: looks like this should be inside the if statement.

File: R/corrplot.R [hunk 3]
```diff
@@ -297,6 +304,26 @@ corrplot <- function(corr,
...
+  # mreg code: replace infinite values by 0,
+  # so that rest of corrplot codes keeps working
+  corr[ is.infinite(corr)] <- 0
```
Comment (vsimko) [inline R/corrplot.R]: also this should be inside the if statement, right ?

**Comment by rlamping** [comment]
Viliam,

 

Good idea. Saves memory usage if not using mreg

 

Met vriendelijke groet / Kind regards

 

Robert Lamping

 

 

-  Ir. R.F. Lamping FC; Havensingel 7; 5611 VR Eindhoven; The Netherlands

(  0031 40 212 1904, ) 0031 6 3 80109 33, *  <mailto:robert@robertlamping.com> robert@robertlamping.com, "  <http://www.robertlamping.com/> www.robertlamping.com  cid:image001.jpg@01CB0BF6.12685980 Robert Lamping

**Comment by rlamping** [comment]
Yep, I will adjust it

 

-  Ir. R.F. Lamping FC; Havensingel 7; 5611 VR Eindhoven; The Netherlands

(  0031 40 212 1904, ) 0031 6 3 80109 33, *  <mailto:robert@robertlamping.com> robert@robertlamping.com, "  <http://www.robertlamping.com/> www.robertlamping.com  cid:image001.jpg@01CB0BF6.12685980 Robert Lamping

File: R/corrplot.R (hunk 2 above)
Comment (rlamping) [reply R/corrplot.R]: Good idea, saves memory when mreg=FALSE

File: R/corrplot.R (hunk 3 above)
Comment (rlamping) [reply R/corrplot.R]: Good idea, saves memory when mreg=FALSE

File: R/corrplot.R (hunk 1 above)
Comment (rlamping) [reply R/corrplot.R]: typo!

**Comment by rlamping** [comment]
Still need to work on the cross size for Inf values and also the width of the colorbar. If additional columns are added, the width of the color bar should not necessarily change.

**Pull request closed without merging**